pkg/logging/logger.go

```diff
@@ -76,32 +76,43 @@ func NewLogger(config *LoggerConfig) *Logger {
 		otelzap.New(
 			log,
 			otelzap.WithMinLevel(zapcore.Level(config.MinLevel)),
-			otelzap.WithTraceIDField(true),
```

Comment:
This option is no longer available in otelzap... Can't find any alternatives and the few entries I find seems to suggest it's not needed. Disagree?